packages/@react-aria/calendar/src/useCalendarCell.ts

```diff
@@ -159,7 +159,7 @@ export function useCalendarCell(props: AriaCalendarCellProps, state: CalendarSta
 
   let isAnchorPressed = useRef(false);
   let isRangeBoundaryPressed = useRef(false);
-  let touchDragTimerRef = useRef(null);
+  let touchDragTimerRef = useRef<ReturnType<typeof setTimeout> | undefined>(undefined);
```

Comment:
makes it easier to call `clearTimeout` because it takes
```
function clearTimeout(timeoutId: NodeJS.Timeout | string | number | undefined): void
```
and null is not one of those, so rather than change
```
clearTimeout(touchDragTimerRef.current ?? undefined);
```
I opted to just change the value we store